server/db/models/index.js

```diff
@@ -7,12 +15,29 @@ const User = require('./user')
  *    BlogPost.belongsTo(User)
  */
 
+Session.hasOne(Order)
+User.hasMany(Session)
+User.hasMany(Review)
+User.hasMany(Order)
+Order.hasMany(OrdersProducts)
```

Comment:
Fix this -- should be many to many and should join through the ordersProducts table